firebase-firestore/src/test/java/com/google/firebase/firestore/spec/SpecTestCase.java

```diff
@@ -132,6 +133,14 @@
   // this tag and they'll all be run (but all others won't).
   private static final String EXCLUSIVE_TAG = "exclusive";
 
+  // The name of a Java system property ({@link System#getProperty(String)}) whose value is the name
+  // of the sole spec test to execute. This is an alternative to setting the {@link #EXCLUSIVE_TAG}
+  // tag, which requires modifying the JSON file. To use this property, specify
+  // -DexclusiveSpecTest=<TestName> to the Java runtime, replacing <TestName> with the name of the
+  // test to execute exclusively.  The <TestName> value is the result of appending the "itName" of
+  // the test to its "describeName", separated by a space character.
+  private static final String EXCLUSIVE_PROPERTY = "exclusiveSpecTest";
```

Comment:
What do you think of making this a filter and allowing Regular Expressions? It should maybe add one or two lines to this PR, but make this feature much easier to use (we don't have to specify the full test name) and more flexible (we could run a group of tests).

Reply:
SGTM.  I'll make this change.